src/main/windows/main.js

```diff
@@ -143,23 +143,15 @@ function setBounds (bounds, maximize) {
   }
 
   // Maximize or minimize, if the second argument is present
-  let willBeMaximized
-  if (maximize === true) {
-    if (!main.win.isMaximized()) {
-      log('setBounds: maximizing')
-      main.win.maximize()
-    }
-    willBeMaximized = true
-  } else if (maximize === false) {
-    if (main.win.isMaximized()) {
-      log('setBounds: unmaximizing')
-      main.win.unmaximize()
-    }
-    willBeMaximized = false
-  } else {
-    willBeMaximized = main.win.isMaximized()
+  if (maximize === true && !main.win.isMaximized()) {
+    log('setBounds: maximizing')
+    main.win.maximize()
+  } else if (maximize === false && main.win.isMaximized()) {
+    log('setBounds: unmaximizing')
+    main.win.unmaximize()
   }
 
+  const willBeMaximized = typeof maximize === 'boolean' ? maximize : main.win.isMaximized
```

Comment:
Just realized that this has a bug and `main.win.isMaximized` needs to be invoked as a function. `main.win.isMaximized()`